makefiles/e2e.mk

```diff
@@ -48,6 +48,7 @@ ADDONSERVER = $(shell pgrep vela_addon_mock_server)
 e2e-apiserver-test:
 	pkill vela_addon_mock_server || true
 	go run ./e2e/addon/mock/vela_addon_mock_server.go &
+	sleep 15
```

Comment:
does it necessary? You know, our CI is too slow now. We should try to make it fast.

Reply:
Yes, the CI must block until addon mock server setup.

Reply:
can we make the wait time less?

Reply:
15 second is more safe otherwise some test-cases may be flaky. Re run all failed test will cause more time.